code/datums/tutorial/marine/medical_basic_item.dm

```diff
@@ -0,0 +1,325 @@
+/datum/tutorial/marine/medical_basic_item
+	name = "Marine - Medical Tutorial 2 (Standard Tools)"
+	desc = "Learn the medical tools you'll need to use to heal yourself and others"
+	tutorial_id = "marine_medical_2"
+	tutorial_template = /datum/map_template/tutorial/s8x9
+
+	var/clothing_items_to_vend = 7
+	var/med_one_items_to_vend = 1
+	var/med_two_items_to_vend = 1
+	var/med_three_items_to_vend = 1
+
+// START OF SCRIPTING
+
+/datum/tutorial/marine/medical_basic_item/start_tutorial(mob/starting_mob)
+	. = ..()
+	if(!.)
+		return
+
+	init_mob()
+	message_to_player("This tutorial will discuss the medical tools you will use as a standard Marine.")
+	addtimer(CALLBACK(src, PROC_REF(intro_one)), 6 SECONDS)
+//pouch_one intro_one
+/datum/tutorial/marine/medical_basic_item/proc/intro_one()
+	message_to_player("You are capable of tending to your own basic injuries without needing Corpsman (sometimes called a Medic) to assist you.")
+	update_objective("")
```

Comment:
You only need to call this when you want to update the objective. No need to set it to a blank string every time.